MNIST/MNIST.swift

```diff
@@ -30,28 +30,46 @@ func readMNIST(imagesFile: String, labelsFile: String) -> (images: Tensor<Float>
     let images = readFile(imagesFile).dropFirst(16).map { Float($0) }
     let labels = readFile(labelsFile).dropFirst(8).map { Int32($0) }
     let rowCount = Int32(labels.count)
-    let columnCount = Int32(images.count) / rowCount
+    let imgRows : Int32 = 28, imgCols : Int32 = 28
 
     print("Constructing data tensors.")
     return (
-        images: Tensor(shape: [rowCount, columnCount], scalars: images) / 255,
+        images: Tensor(shape: [rowCount, 1, imgRows, imgCols], scalars: images)
+		              .transposed(withPermutations: [0, 2, 3, 1]) / 255, // TF: NHWC
```

Comment:
* Indent from the expression by 2 from the previous line. See [Google Swift Style Guide - Line Wrapping (#4)](https://google.github.io/swift/#line-wrapping).

    > A continuation line that is part of a vertically-oriented comma-delimited list is indented exactly +2 from the original line.

* It looks like you used tabs instead of whitespaces on this line.

```suggestion
          .transposed(withPermutations: [0, 2, 3, 1]) / 255, // NHWC
```

Reply:
tweak

Reply:
Ah, I had a typo--it should be indented by 4. We following the Google Swift Style Guide except that all indentations take 4 columns. I'll fix that for you.